jcapiv1/docs/SystemusersApi.md

```diff
@@ -141,7 +136,7 @@ Name | Type | Description  | Notes
 
 <a name="systemusersList"></a>
 # **systemusersList**
-> Systemuserslist systemusersList(contentType, accept, limit, skip, sort, fields, filter)
```

Comment:
Just double checking that this was an expected change

Reply:
limit / skip / sort aren't supported traits so I removed them from Docs.

Reply:
This one removes `contentType` and `accept` from systemusers list